test/ContractTestContext.ts

```diff
@@ -11,7 +11,7 @@ import type {
     PositionToken,
     ParameterStore,
 } from "../types/contracts";
-import type { LvUSDToken } from "../types/contracts/LvUSDToken";
+import type { LvUSDToken } from "../types/contracts/LvUsdToken.sol";
```

Comment:
@CoreyMartin I'm not sure if this change is  needed locally only for my mac or overall, please advise